tools/batch_run_pipeline_qc_face_script.pl

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/perl -w
+
+=pod
+
+=head1 NAME
+
+batch_run_pipeline_qc_face_script.pl -- Run C<pipeline_qc_deface.pl> in batch mode
+
+=head1 SYNOPSIS
+
+perl batch_run_pipeline_qc_face_script.pl [-profile file] [-out_basedir directory] < list_of_files.txt
+
+Available options are:
+
+-profile: name of config file in ../dicom-archive/.loris_mri (typically called prod)
+
+-out_basedir: path to the output base directory where the jpg will be created
+
+=head1 DESCRIPTION
+
+This script runs the creation of 3D rendering QC images on multiple MINC files.
+The list of MINC files to use to generate those 3D JPEG images are provided
+through a text file (e.g. C<list_of_files.txt> with one file path per line.
+
+An example of what a C<list_of_files.txt> might contain for 3 files to use to
+create a 3D JPEG rendering of a scan:
+to be defaced:
+ /data/project/data/assembly/123456/V01/mri/processed/MINC_deface/project_123456_V01_t1w_001_t1w-defaced_001.mnc
+ /data/project/data/assembly/123456/V01/mri/processed/MINC_deface/project_123456_V01_t1w_002_t1w-defaced_001.mnc
+ /data/project/data/assembly/123456/V01/mri/processed/MINC_deface/project_123456_V01_t2w_001_t2w-defaced_001.mnc
+
+
+=head1 LICENSING
+
+License: GPLv3
+
+=head1 AUTHORS
+
+LORIS community <loris.info@mcin.ca> and McGill Centre for Integrative Neuroscience
+
+=cut
+
+
+use strict;
+use warnings;
+no warnings 'once';
```

Comment:
Why the `no warnings 'once'` here?

Reply:
to get rid of the message "variable is used only once" when running the script